docs/en/quickstart/go_sdk.md

```diff
@@ -0,0 +1,104 @@
+# Go SDK Quickstart
+
```

Comment:
specify the requirements: minimal openmdlb version, required components. 

Which ones of go database methods are supported, which is not.